test/test_binary_ufuncs.py

```diff
@@ -2664,6 +2664,11 @@ def test_x(sizes, dim, x, device):
         test_x((1, 10), 0, [1.0], device)
         test_x((0, 2), 0, [], device)
         test_x((0, 2), 1, [1.0, 2.0], device)
+        test_x((2, 3, 4), -1, [1.0, 2.0, 3.0, 4.0], device)
```

Comment:
Would these tests have failed previously?

Reply:
Yes, those are similar to the cases that are failing in issue #63608